src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/CompositionalMultiphaseFVM.hpp

```diff
@@ -63,19 +63,28 @@ class CompositionalMultiphaseFVM : public MGRStrategyBase< 2 >
 
     setupLabels();
 
-    m_levelFRelaxMethod[0]     = MGRFRelaxationMethod::singleLevel; //default, i.e. Jacobi
-    m_levelInterpType[0]       = MGRInterpolationType::jacobi;
-    m_levelRestrictType[0]     = MGRRestrictionType::injection;
-    m_levelCoarseGridMethod[0] = MGRCoarseGridMethod::galerkin;
-
-    m_levelFRelaxMethod[1]     = MGRFRelaxationMethod::singleLevel; //default, i.e. Jacobi
-    m_levelInterpType[1]       = MGRInterpolationType::injection;
-    m_levelRestrictType[1]     = MGRRestrictionType::injection;
-    m_levelCoarseGridMethod[1] = MGRCoarseGridMethod::cprLikeBlockDiag;
-
-    // ILU smoothing for the system made of pressure and densities (except the last one)
-    m_levelSmoothType[1]  = 16;
-    m_levelSmoothIters[1] = 1;
+#ifdef GEOSX_USE_HYPRE_CUDA // Why a different relaxation type for CPU and GPU?
```

Comment:
`MGRFRelaxationType::jacobi` would ask for a Jacobi implementation in hypre that is not GPU-enabled (maybe that's the reason l1jac is chosen instead)?

In the same line, I propose using `7` for Jacobi, this applies also for BoomerAMG. Option `7` runs on CPUs and GPUs

Reply:
Thanks Victor, could you clarify what option `7` is? _hypre_'s documentation is not reporting anything:

https://github.com/hypre-space/hypre/blob/64604ec07539dc41171aad0a97976977d71d5e78/src/parcsr_ls/HYPRE_parcsr_ls.h#L728-L754

Reply:
It's Jacobi as well, but implemented in a different way than option 0. In addition, option 7 supports GPUs. Rui Peng has a branch updating the documentation section you mentioned

Reply:
Perfect, thank you. Then it seems we can always choose option `7`, so no need to distinguish whether we're running on CPU of GPU here:

https://github.com/GEOS-DEV/GEOS/blob/991327d913654ce589c80b17c58ea6d8146cc7b6/src/coreComponents/linearAlgebra/interfaces/hypre/HypreUtils.hpp#L311-L315

https://github.com/GEOS-DEV/GEOS/blob/991327d913654ce589c80b17c58ea6d8146cc7b6/src/coreComponents/linearAlgebra/interfaces/hypre/HypreUtils.hpp#L441-L445

Reply:
Agreed! Using 7 always is a good idea